internal/v2/tls_config_store/tls_config_store.go

```diff
@@ -219,7 +218,6 @@ func getTLSMinMaxVersionsClient(tlsConfig *s2av2pb.GetTlsConfigurationResp_Clien
 	return minVersion, maxVersion, nil
 }
 
-// TODO(rmehta19): Test every possible case of input/outputs for this function.
 func getTLSMinMaxVersionsServer(tlsConfig *s2av2pb.GetTlsConfigurationResp_ServerTlsConfiguration) (uint16, uint16, error) {
 	// Map S2Av2 TLSVersion to consts defined in tls package.
 	var minVersion uint16
```

Comment:
We should refactor the 4 switch statements to a helper function that takes the tlsConfig version and returns the tls.Versions. This switch statements can then be replaced with calls to this helper function. We can do this in a separate PR. Thanks!

Reply:
Will do! I added a TODO.